tools/workspace-plugin/src/generators/version-bump/index.spec.ts

```diff
@@ -349,3 +349,13 @@ function setupDummyPackage(
 
   return tree;
 }
+
+function assertAndReadNxJson(tree: Tree) {
```

Comment:
This function is repeated in every test file, maybe export it from one of the utility files for reuse?

Reply:
yeah I wanted to do that initially but wasn't 100% sure it is generic enough :D, so I decided to keep duplicated it for now